Loop54.Shared/Model/Request/GetBasketRecommendationsRequest.cs

```diff
@@ -0,0 +1,31 @@
+using Loop54.Model.Request.Parameters;
+using System;
+using System.Collections.Generic;
+
+namespace Loop54.Model.Request
+{
+    /// <summary>
+    /// Used to perform a request to get recommendations for a basket.
+    /// </summary>
+    public class GetBasketRecommendationsRequest : Request
+    {
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="entities">The entities to use in the query.</param>
```

Comment:
Nit: this param should probably have the same wording as `Entities`, so no one is confused that they might not be the same.